ui/src/sd-admin/source.tsx

```diff
@@ -0,0 +1,33 @@
+import { StudiesApiContext } from "apiContext";
+import { useContext, useMemo } from "react";
+import * as tanagra from "tanagra-api";
+import { CreateStudyRequest, StudyV2, UpdateStudyRequest } from "tanagra-api";
+
+export interface AdminSource {
```

Comment:
Created a new source file since we're keeping the sd admin stuff separate. I can move these to the existing file if we think these endpoints will also be needed in core Tanagra.

Reply:
I think this is the right choice for now.